### PR TITLE
create Entity breed

### LINKED ISSUE #2: Create Entity Breed
-> `\src\Domain\Entity\Breed.php`

- id (uuid)
- name (string, 50)
- description (string, 255)

### PR DESCRIPTION
**Pull request by Alves7777**
Resolve #2

### CODE REVIEW

File: src/Entity/Breed.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Uid\Uuid;
+
+/**
+ * @ORM\Entity()
+ */
+class Breed
+{
+    /**
+     * @ORM\Id()
```
Comment (mariaclaraabreu) [inline src/Entity/Breed.php]: not need

File: src/Entity/Breed.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+     * @ORM\Column(type="integer")
```
Comment (mariaclaraabreu) [inline src/Entity/Breed.php]: Should be type="uuid"

File: src/Entity/Breed.php [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+     * @ORM\GeneratedValue()
```
Comment (mariaclaraabreu) [inline src/Entity/Breed.php]: not need

File: src/Entity/Breed.php [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+     */
+    private Uuid $id;
+
+    /**
+     * @ORM\Column(type="string", length=50)
+     */
+    private string $name;
+
+    /**
+     * @ORM\Column(length=100, length=100)
```
Comment (alessandrofeitoza) [inline src/Entity/Breed.php]: the attribute `length` are duplicated, and the length correct is 255.

File: src/Entity/Breed.php [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+     */
+    private string $description;
+
+
+    public function getId(): uuid
```
Comment (alessandrofeitoza) [inline src/Entity/Breed.php]: change `uuid` for `Uuid`

File: src/Entity/Breed.php [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+    {
+        return $this->id;
+    }
+
+    public function setId(uuid $id): void
```
Comment (alessandrofeitoza) [inline src/Entity/Breed.php]: change `uuid` for `Uuid`, because is a class.

File: src/Entity/Breed.php [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+    /**
+     * @ORM\Column(type="string", length=50)
+     */
+    private string $name;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private string $description;
+
```
Comment (mariaclaraabreu) [inline src/Entity/Breed.php:29]: remove empty line

**Review by alessandrofeitoza — APPROVED** [review]

**Review by pamplona007 — APPROVED** [review]

**Pull request merged**